avocado/utils/distro.py

```diff
@@ -304,7 +304,7 @@ def get_distro(self):
             self.score += 1
 
         if self.check_release():
-            release = self.release()
+            release = self.release() or UNKNOWN_DISTRO_RELEASE
```

Comment:
This is needed, because I used optional group `(\d{1,2})?` for the release. But when I am looking at it now, I think the better place will be in `self.release()`. I will send an update.